src/main/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/config/ErrorCodes.kt

```diff
@@ -29,15 +29,16 @@ object MalformedRequest : StandardErrorCodes("MALFORMED_REQUEST", "Failed to rea
 object NotFound : StandardErrorCodes("NOT_FOUND", "Not found")
 
 @Schema(oneOf = [EmailMandatory::class, EmailTooLong::class, EmailInvalid::class, EmailInvalidCjsm::class])
-sealed class MagicLinkRequestErrorCodes(
+sealed class AuthenticationRequestErrorCodes(
```

Comment:
I've renamed this so I didnt have to create duplicate error code classes.
I'll need to re-import the types into the UI, but I'm going to have to do that anyway for the new endpoint and associated request body type